alerts/openshift-virtualization-operator/KubeVirtDeprecatedAPIsRequested.md

```diff
@@ -0,0 +1,29 @@
+# KubeVirtDeprecatedAPIsRequested
+
+## Meaning
+
+This alert fires when a deprecated KubeVirt API is used.
+
+## Impact
+
+Using a deprecated API is not recommended because the request will
+fail when the API is removed in a future release.
+
+## Diagnosis
+
+- Check the `description` and `summary` alert annotations to identify the
+deprecated API:
+
+  ```text
+                description: "Detected requests to the deprecated virtualmachines.kubevirt.io/v1alpha3 API."
+                summary: "2 requests were detected in the last 10 minutes."
+  ```
+
+## Mitigation
+
+Use fully supported APIs. The alert resolves itself after 10 minutes if the deprecated
+API is not used.
+
+Note: If an alert is raised by an OpenShift Virtualization component using a
+deprecated API, you can ignore the alert. The alert will resolve itself when the
+API is removed in a future release.
```

Comment:
How can the user know if the request to the deprecated API was originated from an internal component?
I think this would require to dig in the audit logs with verbosity to find it, and it is not something very visible to the end user.

Anyway, I don't think deprecated APIs are being used internally in the same release those APIs are being deprecated, and if so, it's a bug, IMHO.

Reply:
I agree. If irrelevant alerts are being raised by components, that's a bug and not something that the user should deal with.

I will remove this paragraph.

Reply:
@0xFelix WDYT?

Reply:
@0xFelix  So , if its not considered in the alert then I think its ok to remove the comment right?